components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -214,6 +233,12 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
           </div>
         </Collapse>
 
+        <Collapse in={this.state.ingress === IngressType.Skip}>
+          <div style={styles.row}>
+            <li>Skip creating service endpoint, use port-forward to connect to kubeflow service</li>
```

Comment:
Can we say something like access IAP via UI but complete OAuth setup later?

I don't want to call out port-forwarding because then we are setting an expectation that this is an option we will support in the future.